cmd/openshift-install/create.go

```diff
@@ -128,3 +140,75 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 		return nil
 	}
 }
+
+// FIXME: pulling the kubeconfig and metadata out of the root
+// directory is a but cludgy when we already have them in memory.
+func destroyBootstrap(ctx context.Context, directory string) (err error) {
+	config, err := clientcmd.BuildConfigFromFlags("", filepath.Join(directory, "auth", "kubeconfig"))
+	if err != nil {
+		return err
+	}
+
+	client, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	discovery := client.Discovery()
+
+	apiContext, cancel := context.WithTimeout(ctx, 30*time.Minute)
+	defer cancel()
+	wait.Until(func() {
+		version, err := discovery.ServerVersion()
+		if err == nil {
+			logrus.Infof("API %s up", version)
+			cancel()
+		}
+		logrus.Debugf("API not up yet: %s", err)
+	}, 2*time.Second, apiContext.Done())
+
+	events := client.CoreV1().Events("kube-system")
+
+	logrus.Info("wait for bootstrap completion")
```

Comment:
"Waiting for bootstrap completion..."

I like to make this a little more human-friendly since this get's printed out during normal operation and isn't consumed by any machines.

Reply:
> "Waiting for bootstrap completion..."

Done in 5db1473 -> c0e0a67, which also rebases onto the current #573 tip.